### PR TITLE
Change default author of exercise to hostname

### PR DESCRIPTION
**Pull request by jamessimas**
Fix for issue #159.

Commit c687301 fixes issue #159 by making the default author for new
exercises the hostname of the local computer running wger.

This commit (b00efde) updates the test to ensure this is what we test for.

### CODE REVIEW

**Comment by rolandgeider** [comment]
I probably wrote the ticket unclearly! I meant to write that the name of the (webserver)host be used, so if someone has an instance running on "my-local-gym.com", that value is used. The socket library reads the local name of the server running the code. I have googled a bit and found that django's request object has a [get_host()](https://docs.djangoproject.com/en/1.7/ref/request-response/#django.http.HttpRequest.get_host) that returns what we need

**Comment by jamessimas** [comment]
Ok, thanks for the clarification!

I'll begin working on fixing this tonight.

**Comment by jamessimas** [comment]
Ok. Done. 

Thoughts?

File: wger/exercises/models.py [hunk 1]
```diff
@@ -330,8 +331,16 @@ def set_author(self, request):
...
             if not self.license_author:
-                self.license_author = 'wger.de'
+                #  Get everything up to semicolon, exclusive.
+                #  Example: my-gym.com:9090 returns my-gym.com
+                results = re.search('^([^:]+).*', request.get_host())
+
+                if results:
+                    name = results.group(1)
+                else:
+                    name = request.get_host()
```
Comment (rolandgeider) [inline wger/exercises/models.py]: Regexes are a bit of overkill here, we can easily extract the part from the hostname that we need with a split: ``get_host().split(':')[0]``

**Comment by jamessimas** [comment]
You're totally right. I'll fix this tonight. 

James

**Comment by jamessimas** [comment]
I've updated my pull request with the change you suggested.

Any idea why it's failing in your Travis CI instance, but not in mine? (https://travis-ci.org/jamessimas/wger/builds/69990038)

I'm heading out of the country for the next few weeks. Internet access is pretty spotty, so I won't get a chance to continue working on this until I return.

**Comment by rolandgeider** [comment]
I am also sometimes getting these errors, it seems it started some days ago. Perhaps it's something on travis' side? In any case, I'll merge this now. 

If you are going on vacation, relax, and don't think about issues! ;)

**Comment by rolandgeider** [comment]
@jamessimas How are you doing? Back from vacation?

**Comment by jamessimas** [comment]
@rolandgeider 

Thanks for checking in. I'm in the process of moving right now so I've been busy. I'll check back as soon as I wrap that up and have more time.

**Comment by rolandgeider** [comment]
Cool, no problem!

**Pull request merged**